lib/tarfiles.py

```diff
@@ -271,30 +271,41 @@ def tarfile_in_dropbox(args: argparse.Namespace, origtfn: str) -> Optional[str]:
         # cid looks something like dune/bf6a15b4238b72f82...(long hash)
         cid = f"{args.group}/{digest}"
         if args.verbose:
-            print(f"Using RCDS to publish tarball\ncid: {cid}")
+            print(f"\n\nUsing RCDS to publish tarball\ncid: {cid}")
 
         publisher = TarfilePublisherHandler(cid, proxy, token)
         location = publisher.cid_exists()
         if location is None:
             publisher.publish(tf)
-            print("Checking to see if uploaded file is published on RCDS.")
-            # pylint: disable-next=unused-variable
-            for i in range(NUM_RETRIES):
-                location = publisher.cid_exists()
-                if location is not None:
-                    print("Found uploaded file on RCDS.")
-                    break
-                if i < (NUM_RETRIES - 1):
+            if not getattr(args, "skip_check_rcds", False):
```

Comment:
So old jobsub uses a shell function in the wrapper:
https://cdcvs.fnal.gov/redmine/projects/jobsub/repository/revisions/master/entry/lib/groupsettings/JobUtils.py#L303
I note that it doesn't actually call the web api, which is probably good, it just polls the possible locations, given the cid. 
It might be cleaner to put a simpler version in the simple.sh wrapper.

Reply:
So it old jobsub ends up looking at, i.e. https://rcds01.fnal.gov/pubapi/config and getting back
`repos:fifeuser1.opensciencegrid.org,fifeuser2.opensciencegrid.org,fifeuser3.opensciencegrid.org,fifeuser4.opensciencegrid.org`
and drops those in  the wrapper and  has a for loop over each of those  looking in /cvms/{repo}/sw/{cid} . 

So to emulate this much more simply if we're punting on the publish check, we should:
* lookup the config entry and get the repo list in repolist
* set location to a glob string like: `f'/cvmfs/\{{",".join(repolist)}\}/sw/{digest}'`
* make sure our loop in simple.sh will work with that glob pattern.

Reply:
Working on a rewrite of this PR that includes all of this.  I realized that a lot of what I did with the skip_checks.py can be cleanly done with an Enum, so I'm redoing it.  I expect it'll be ready by the end of the week.

Reply:
Almost done with this.  Just need to run some test.  I'm going to put this in Draft, and then mark it Ready when it's....Ready.